dts/bindings/display/led-strip-matrix.yaml

```diff
@@ -0,0 +1,181 @@
+        [ 7][ 6][ 5][ 4]  [23][22][21][20]
+        [ 8][ 9][10][11]  [24][25][26][27]
+        [15][14][13][12]  [31][30][29][28]
+
+  vertical-modules:
+    type: int
+    default: 1
+    description: |
+      If the display forms with multiple modules,
+      specify the vertical number of modules.
+      The number must be able to divide the height value.
+      If not set, it controls a single matrix.
+
+      * 4x8 display with 2 serpentine layout modules
+        [ 0][ 1][ 2][ 3]
+        [ 7][ 6][ 5][ 4]
+        [ 8][ 9][10][11]
+        [15][14][13][12]
+
+        [16][17][18][19]
+        [23][22][21][20]
+        [24][25][26][27]
+        [31][30][29][28]
+
+  modules-circulative:
+    type: boolean
+    description: |
+      Specifies that the order of the modules that make up the matrix is circulative.
+
+      * circulative module layout
+        [M0][M1][M2]
+        [M3][M4][M5]
+        [M6][M7][M8]
+
+      * serpentine module layout
+        [M0][M1][M2]
+        [M5][M4][M3]
+        [M6][M7][M8]
+
+  modules-start-from-right:
+    type: boolean
+    description: |
+      Specifies that modules are ordered from right to left.
+
+      * Start from the right with a module serpentine layout
+        [M2][M1][M0]
+        [M3][M4][M5]
+        [M8][M7][M6]
+
+      * Start from the right with a module circulative layout
+        [M2][M1][M0]
+        [M5][M4][M3]
+        [M8][M7][M6]
+
+  modules-start-from-bottom:
+    type: boolean
+    description: |
+      Specifies that modules are ordered from bottom to top.
+
+      * Start from the right with a module serpentine layout
+        [M6][M7][M8]
+        [M5][M4][M3]
+        [M0][M1][M2]
+
+      * Start from the right with a module circulative layout
+        [M6][M7][M8]
+        [M3][M4][M5]
+        [M0][M1][M2]
+
+  led-strips:
+    type: phandles
+    required: true
+    description: |
+      Specify the LED strip that is the substance of the matrix.
+      If multiple strips are specified, they are "flattened" and sequentialized.
+      For example, if `strip0` and `strip1` with 128 LEDs are specified,
+      the first LED of `strip1` will be treated as the 129th LED.
+      These LEDs are mapped to coordinates according to the layout rule in order.
+      The amount of LEDs must equal the [width * height] value.
+
+  chain-lengths:
+    type: array
+    description: |
+      Specify the number of LEDs for each strip.
+      It can omit the value if all strip nodes have a `chain-length` property.
+      Each value must be a multiple of the number of LEDs per module
+      [(width / horizontal-modules) * (height / vertical-modules)].
+
+  pixel-format:
+    type: int
+    description: |
+      Initial Pixel format.
+      See dt-bindings/display/panel.h for a list.
+      This property only accept PANEL_PIXEL_FORMAT_RGB_888 and PANEL_PIXEL_FORMAT_RRGB_8888.
+      If not set this property, the behavior is same as selecting PANEL_PIXEL_FORMAT_RGB_888.
```

Comment:
```suggestion
      This property only accepts PANEL_PIXEL_FORMAT_RGB_888 and PANEL_PIXEL_FORMAT_RRGB_8888.
      If this property is not set, the behavior is same as selecting PANEL_PIXEL_FORMAT_RGB_888.
```

Reply:
btw why not just use PANEL_PIXEL_FORMAT_RGB_888 (1) as a default value here?

Reply:
I didn't deeply consider it. It's easier to understand if you explicitly set the default value, so I've modified it that way.